test/controllers/ReferenceDataControllerSpec.scala

```diff
@@ -45,39 +49,62 @@ class ReferenceDataControllerSpec extends SpecBase with GuiceOneAppPerSuite with
   val mockReferenceDataService    = mock[ReferenceDataService]
   val mockSchemaValidationService = mock[SchemaValidationService]
 
-  private val testJson = Json.obj("foo" -> "bar")
+  def compress(input: Array[Byte]): Array[Byte] = {
+    val bos  = new ByteArrayOutputStream(input.length)
+    val gzip = new GZIPOutputStream(bos)
+    gzip.write(input)
+    gzip.close()
+    val compressed = bos.toByteArray
+    bos.close()
+    compressed
+  }
```

Comment:
Is this used in 2 specs? Maybe consider putting it in spec base (looks like the same implementation used in both places)